openmetrics/README.md

```diff
@@ -12,29 +12,29 @@ The integration is compatible with both the [Prometheus exposition format][12] a
 
 Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][1] for guidance on applying these instructions.
 
+This OpenMetrics-based integration has a latest mode (`use_openmetrics`: true) and a legacy mode (`use_openmetrics`: false). To get all the most up-to-date features, Datadog recommends enabling the latest mode. For more information, see [Latest and Legacy Versioning For OpenMetrics-based Integrations][15].
```

Comment:
`openmetrics_endpoint` rather than `use_openmetrics`, and I think it might make more sense to drop the "OpenMetrics-based" part for this one since we are talking about *the* OpenMetrics integration.

```suggestion
This integration has a latest mode (enabled by setting `openmetrics_endpoint` to point to the target endpoint) and a legacy mode (enabled by setting `prometheus_url` instead). To get all the most up-to-date features, Datadog recommends enabling the latest mode. For more information, see [Latest and Legacy Versioning For OpenMetrics-based Integrations][15].
```